attributes/default.rb

```diff
@@ -68,9 +68,11 @@
 
 # Configuration for Windows scheduled task
 default['chef_client']['task']['frequency'] = 'minute'
-default['chef_client']['task']['frequency_modifier'] = node['chef_client']['interval'].to_i / 60
+default['chef_client']['task']['frequency_modifier'] = 30
 default['chef_client']['task']['user'] = 'SYSTEM'
-default['chef_client']['task']['password'] = nil # Password is only required for none system users
+default['chef_client']['task']['password'] = '' # Password is only required for none system users
+default['chef_client']['task']['task_name'] = 'chef-client'
+default['chef_client']['task']['start_time'] = '06:30'
```

Comment:
Is there any reason to update these attributes for fixing https://github.com/chef-cookbooks/chef-client/issues/490?